sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -962,4 +962,20 @@ private void validateUserType(Request userRequest) {
               new String[] {userType, JsonKey.USER_TYPE}));
     }
   }
+
+  public void validateUserMergeRequest(Request request) {
+    if(null == request.getRequest().get("mergeeId") || StringUtils.isEmpty((String) request.getRequest().get("mergeeId"))) {
```

Comment:
Use StringUtils.IsBlank this will check both emty and null

"mergeeId"  Use constant instead of hardcode value

Reply:
Used StringUtils.IsBlank, it serving both.